anyblok/migration.py

```diff
@@ -659,6 +709,47 @@ def drop(self):
         return self
 
 
+class MigrationColumnTypePlugin:
+    """Meta class for column migration type plugin
+
+    Must be exposed as entry point in namespace 'anyblok.migration_type.plugins'
+
+    :param to_type: New column type value (sqlalchemy.types)
+    :param from_type: Old column type value (sqlalchemy.types)
```

Comment:
I feel `to` and `from as `new` and `old` are no such accurate to avoid ambiguous.

Could you at least change doctstring and maybe attribute names ?

I would suggest following description
```suggestion
    :param to_type: Column type value (sqlalchemy.types) as used in Model classes in source code
    :param from_type: Column type value (sqlalchemy.types) as required to communicate with the DBMS
```

as long it's for migration from/to is quite ok could also change to `code_type` and `sgbd_type` but won't ask to change on this topic and I'm not sure to understand properly this issue